.zuul.yaml

```diff
@@ -198,6 +196,18 @@
       cifmw_update_containers_openstack: true
       cifmw_update_containers_org: podified-master-centos9
 
+##########################################################
+#                                                        #
+#               Zuul Pragma to alter Zuul Config         #
+#                                                        #
+##########################################################
+
+- pragma:
+    implied-branch-matchers: True
+    implied-branches:
+      - master
+      - main
```

Comment:
we should add a note that we need to delete it every time we branch for FR*

if we don't do that when we create the FR branches there job definitions will break main.

Reply:
Done!